loqusdb/utils/load.py

```diff
@@ -199,18 +203,22 @@ def load_variants(
     max_window=3000,
     variant_type="snv",
     genome_build=None,
+    ignore_gq_if_unset=False,
 ):
     """Load variants for a family into the database.
 
     Args:
         adapter (loqusdb.plugins.Adapter): initialized plugin
+        vcf_obj(cyvcf2.VCF): Iterable with cyvcf2.Variant
```

Comment:
Thank you! Consider typehinting instead of filling up old args list, but dont feel you have to.